.docker/entrypoint/Gemfile

```diff
@@ -19,6 +19,7 @@ gem 'faraday', '~> 1.0.1'
 gem 'html-proofer'
 gem 'html-proofer-unrendered-markdown'
 gem 'rake', '~> 13.0'
+gem 'concurrent-ruby'
```

Comment:
Codacy found an issue: [Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `concurrent-ruby` should appear before `rake`.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6067254)